Source/d_main.c

```diff
@@ -2356,7 +2360,7 @@ void D_DoomMain(void)
         {
           I_StartTic ();
           D_ProcessEvents ();
-          G_BuildTiccmd (&netcmds[consoleplayer]);
+          G_BuildTiccmd (netcmds);
```

Comment:
I don't get why timedemo was broken before this change? Could you explain?

Reply:
In MBF `netcmds` is initialized as a static global array, in Choco it's just a pointer. Choco has `ticdata` which is similar to MBF's `netcmds`:
https://github.com/fabiangreffrath/woof/blob/1f2c476af169cf07fea04149f2cb19ad42d92fd7/Source/d_loop.c#L64

Reply:
Also `G_BuildTiccmd(ticcmd_t* cmd)` initialize `cmd` parameter with this code:
https://github.com/fabiangreffrath/woof/blob/1f2c476af169cf07fea04149f2cb19ad42d92fd7/Source/g_game.c#L288-L289
"Historical" code I guess :smile:

Reply:
Thank you!

Reply:
Why do we need `G_BuildTiccmd` in `singletics` mode? We have no game input. Is this for a more honest FPS measurement?